tests/guest-tools/unix/test_guest_tools_unix.py

```diff
@@ -15,7 +15,9 @@ def __init__(self):
         self.tools_version = None
         self.vm_distro = None
 
-@pytest.mark.incremental
+@pytest.mark.incremental # tests depend on each other. If one test fails, don't execute the others
+@pytest.mark.multi_vms # meant to run on a variety of VMs
+@pytest.mark.usefixtures("unix_vm") # requires unix VMs
```

Comment:
Because these marks are described in `pytest.ini`, I'm not sure it's useful and maintainable to duplicate the definition of each mark when they are used.